.github/workflows/pluto_build.yml

```diff
@@ -84,9 +71,8 @@ jobs:
           rm -rf "$AGENT_TOOLSDIRECTORY"
```

Comment:
✂️ ? Figure we're not trying to save this space anymore

If not this, I would definitely say ✂️  lines 97-98 (clean database job)

Reply:
snipped in [05a4603](https://github.com/NYCPlanning/data-engineering/pull/303/commits/05a46032f7fe5320756f1a35ab8b4b7e2c7ede19)